src/nixos/machines/ignucius/config/vm-build.nix

```diff
@@ -0,0 +1,24 @@
+{ ... }:
+
+# VM configuration of IGNUCIUS, used for testing prior to deployment
+
+{
+	# FIXME(Krey): Neither of those are working right now, see https://github.com/nix-community/disko/issues/668
+	virtualisation = {
```

Comment:
This has been fixed -> Adjust the merge request to have the ability to test changes in VM prior to deployment

Reply:
Tracked in https://github.com/NiXium-org/NiXium/issues/148